bundle/manifests/0.7.0/skupper-operator.v0.7.0.clusterserviceversion.yaml

```diff
@@ -23,12 +23,12 @@ spec:
   replaces: skupper-operator.v0.6.0
   apiservicedefinitions: {}
   description: |
-    Skupper site operator.
+    Skupper operator.
 
 
-    The Skupper site controller operator helps you managing your site by watching for a
+    The Skupper operator helps you managing your site by watching for a
```

Comment:
depending on where this text appears, it might be necessary to explain what a site is.

Reply:
It is displayed in the Operator Hub. Do you remember if we already have a definition of 
what a Skupper site is in the documentation? I looked at the documentation and I could
not find any standardized description.

Maybe instead of just explaining what a Skupper site is, I believe we could introduce what
Skupper does for you and how the operator can help you.

Today it says:
```
The Skupper site controller operator helps you managing your site by watching for a ConfigMap named skupper-site. To get started with it you can simply define this ConfigMap in the desired namespace and the skupper operator will take care of defining your site accordingly.
```

Any recommendation? Maybe something that introduces Skupper itself first, then another paragraph explaining the operator role, like:

```
Skupper helps you enabling communication across clusters through its service network.

The Skupper operator helps you managing your site (a Skupper instance that runs in a namespace),
watching for the presence of a ConfigMap named skupper-site in your namespace. The operator can
be installed to a single namespace or at cluster level, watching for skupper-site config maps in
all namespaces.

To get started with it you can simply define the following ConfigMap in the desired namespace and the
Skupper operator will take care of defining your site accordingly.
```

Reply:
I like your idea, how about:

Skupper enables communication between services running in different network locations. It allows geographically distributed services to connect as if they were all running in the same site.

Each site is a namespace with a running Skupper instance.

The Skupper operator allows you create and manage sites.
The operator processes any ConfigMap named skupper-site and configures the site for the service network accordingly. 

To get started, create the following ConfigMap in the desired namespace, the
Skupper operator will create the Skupper instance and configure the site.

...

The operator can be installed to a single namespace or at cluster level, watching for the skupper-site ConfigMap in all namespaces.

Reply:
Much better to me :) 
@ajssmith @grs @nluaces any thoughts?

Reply:
+1

Reply:
+1 !